### ISSUE TITLE
First Open Camera Picker and after Press Android hardware back button Will Restart App

### ISSUE THREAD

**Issue by komelabbbas** [ivpusic/react-native-image-crop-picker#1490]
### Version

- react-native-image-crop-picker v0.35.2
- react-native v0.63.2

### Platform
Android

Any Solution..?
plz help if another one fix already.

**Comment by komelabbbas** [comment]
Any update..?

**Comment by Anujmoglix** [comment]
any update ?

**Comment by superandrew213** [comment]
Can you turn off all other apps that are in the background and test again?

**Comment by komelabbbas** [comment]
> Can you turn off all other apps that are in the background and test again?

yes i have already turned off all other apps and check but not work!.
My Device Is Android Redmi 7A.

plz help if another one fix.
